vendor/knative.dev/hack/library.sh

```diff
@@ -512,6 +512,7 @@ function go_update_deps() {
 
   export GO111MODULE=on
   export GOFLAGS=""
+  export GOSUMDB=off   # Do not use the sum.golang.org service.
```

Comment:
Is this what we want? 
/cc @slinkydeveloper

Reply:
:man_shrugging:

Reply:
/hold

Reply:
Maybe we should ask to @n3wscott ?

Reply:
We had a thread in productivity.